pom.xml

```diff
@@ -10,21 +10,26 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>azure-container-registry-tasks-plugin</artifactId>
-    <version>0.4.0</version>
+    <version>0.5.0-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>1.651.3</jenkins.version>
         <java.level>7</java.level>
 
-        <azuresdk.version>1.17.2-SNAPSHOT</azuresdk.version>
+        <jenkins.version>1.651.3</jenkins.version>
+        <jenkins-scm-api.version>1.3</jenkins-scm-api.version>
+
+        <azure-sdk.version>1.18.0</azure-sdk.version>
```

Comment:
should this be added to azure-common repo also for central azure dependency management?

Reply:
Not necessary. The plugin and common plugin both depend on azure sdk, but they use it for different purposes. Azure-common mainly use azure sdk for credential related work.

Reply:
@yuwzho  do you remember the classpath loading trap Menghua shared?

Reply:
@yungezz Classloader issue is hpi plugin will pack all the dependencies and depth-first to load the class. So we use maskClasses to exclude the package we don't want to pack.

https://github.com/Azure/azure-acr-plugin/blob/06b71d9c77a02e1c15cb97732a4aa7724b39011b/pom.xml#L218-L223